Description: Fix bidirectional TCP server exiting after the test
 iperf bidirectional test using parameter -r or -d doesn't work as
 expected, the server exits after the test, but it should continue listening
 for new connections.
Author: Roberto Lumbreras <rover@debian.org>
Bug-Redhat: https://bugzilla.redhat.com/show_bug.cgi?id=449796
Forwarded: http://sourceforge.net/tracker/?func=detail&aid=1983829&group_id=128336&atid=711371

--- a/src/Client.cpp
+++ b/src/Client.cpp
@@ -212,10 +212,12 @@ void Client::Run( void ) {
     char* readAt = mBuf;
 
 #if HAVE_THREAD
+    /*
     if ( !isUDP( mSettings ) ) {
 	RunTCP();
 	return;
     }
+    */
 #endif
     
     // Indicates if the stream is readable 
